StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentMethodsView.swift

```diff
@@ -213,7 +205,84 @@ class EmbeddedPaymentMethodsView: UIView {
     }
 
     func didTapAccessoryButton() {
-        // TODO(porter)
+        guard let savedPaymentMethod = savedPaymentMethods.first else {
+            stpAssertionFailure("Tapped on accessory button but there are no saved payment methods")
+            return
+        }
+        delegate?.presentSavedPaymentMethods(selectedSavedPaymentMethod: savedPaymentMethod)
+    }
+
+    func updateSavedPaymentMethods(_ savedPaymentMethods: [STPPaymentMethod],
+                                   userSelectedPaymentMethod: STPPaymentMethod?,
```

Comment:
What is `userSelectedPaymentMethod`?

Reply:
I may need to re-think this. I was using it to decide whether or not to update the selected payment method.  I'll revisit the code and figure out if this is needed.

Reply:
I believe this is needed for the use case of when a user selects a saved payment method, and we want to update the main view so that it updates the selection.  It was incorrectly being passed in for the 'update' case, which has been updated.